integration_test/deployment-cumulus-orca.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 set -ex
 
-export CUMULUS_AWS_ACCESS_KEY_ID=$bamboo_CUMULUS_AWS_ACCESS_KEY_ID
-export CUMULUS_AWS_SECRET_ACCESS_KEY=$bamboo_CUMULUS_AWS_SECRET_ACCESS_KEY
+export AWS_ACCESS_KEY_ID=$bamboo_CUMULUS_AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$bamboo_CUMULUS_AWS_SECRET_ACCESS_KEY
```

Comment:
Got errors unless I used values AWS TF would automatically pick up.

Reply:
somehow CUMULUS prefix got added to these variables plus AWS_DEFAULT_REGION in some PR